src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { Dispatch, Fragment, SetStateAction } from 'react';
+import cn from 'classnames';
+import { v4 as getId } from 'uuid';
+
+import './Pagination.scss';
+import { ProductItem } from '../ProductItem/ProductItem';
+import { Product } from '../../types/Product';
+
+type Props = {
+  productsLength: number
+  itemsPerPage: number,
+  currentPage: number,
+  setCurrentPage: Dispatch<SetStateAction<number>>,
+  products: Product[],
+};
+
+export const Pagination: React.FC<Props> = ({
+  productsLength,
+  itemsPerPage,
+  currentPage,
+  setCurrentPage,
+  products,
+}) => {
+  const pages = [];
+  const lastPage = Math.ceil(productsLength / itemsPerPage);
+
+  for (let i = 1; i <= lastPage; i += 1) {
+    pages.push(i);
+  }
+
+  const handleNextClick = () => {
+    setCurrentPage(i => {
+      if (i >= lastPage) {
+        return lastPage;
+      }
+
+      return i + 1;
+    });
+  };
+
+  const handlePrevClick = () => {
+    setCurrentPage(i => {
+      if (i <= 1) {
+        return 1;
+      }
+
+      return i - 1;
+    });
+  };
+
+  return (
+    <div className="pagination">
+      <div className="pagination__list">
+        {products.map(phone => (
+          <Fragment key={getId()}>
+            <ProductItem
```

Comment:
It's unclear from `Pagination` component that it contains the list of products.

`Pagination` component should contain only pagination:
![image](https://github.com/mate-academy/react_phone-catalog/assets/86923089/3ffcbd8b-991a-4489-bba4-b26a9004fdd5)